src/main/java/ir/sahab/kafkaconsumer/OffsetTracker.java

```diff
@@ -122,12 +122,21 @@ public PartitionTracker(long initialOffset) {
         boolean track(long offset) {
             long pageIndex = offsetToPage(offset);
             int margin = (int) (offset % pageSize);
```

Comment:
```suggestion
            int pageOffset = (int) (offset % pageSize);
```

Actually it is pageOffset and if it is used as the argument to PageTracker constructor, this offset indicates the margin too. So it is better to name it pageOffset. (As we have done in ack() method)